web_editor_disable_chatgpt/tests/test_tours.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import odoo.tests
+
+
+@odoo.tests.tagged("post_install", "-at_install")
+class TestTours(odoo.tests.HttpCase):
+    def test_disable_chatgpt(self):
+        self.start_tour(
```

Comment:
I could do something like this, but it could potentially hide legitimate browser warnings in the future.

```suggestion
        # Ignore console.warn Missing widget: res_partner_many2one for field of type many2one
        with mute_logger("odoo.addons.web_editor_disable_chatgpt.tests.test_tours.TestTours.test_disable_chatgpt.browser"):
            self.start_tour(
```